cookbook/how_to_install_extension_from_command_line.rst

```diff
@@ -82,7 +82,7 @@ Finally, make sure to properly clean the cache:
 Querying Extension Information
 ------------------------------
 
-There are two commands that help you to get an overview about the extension.
+There are two commands that help you get an overview about the extension.
```

Comment:
Thanks for the link @nnenasheva.

@akuzmenko-magecore You can ignore my comment then. :)